### PR TITLE
Closes #160 CI/CD checks failing: Links/Validate Links and Roxygen check

### LINKED ISSUE #160: General Issue: CI/CD checks failing
### Background Information

# Links / Validate Links
### Errors in .github/pull_request_template.md

* [404] <https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process> | Rejected status code: 404 Not Found (configurable with "accept" option)

# Roxygen Check

💻 Please rerun the following command on your workstation and push your changes
-------------------------
roxygen2::roxygenize('.')
-------------------------
ℹ roxygen2 version that was used in this workflow: ‘8.0.0’
🙏 Please ensure that the 'RoxygenNote' field in the DESCRIPTION file matches this version

### Definition of Done

# Links / Validate Links

Update the link in `github/pull_request_template.md` from `https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process` to `https://pharmaverse.github.io/admiral/cran-release/CONTRIBUTING.html#detailed-development-process`

# Roxygen Check

💻 Please rerun the following command on your workstation and push your changes
-------------------------
roxygen2::roxygenize('.')
-------------------------

### PR DESCRIPTION
**Pull request by Fanny-Gautier**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build pharmaverseadam site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new datasets occur on the "[Reference](https://pharmaverse.github.io/pharmaverseadam/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -59,4 +59,4 @@ Language: en-US
 LazyData: true
 LazyDataCompression: bzip2
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.3.3
+Config/roxygen2/version: 8.0.0
```
Comment (manciniedoardo) [inline DESCRIPTION:62]: Do you need more fields here for the upversioning to roxygen 8.0.0? see for example [here](https://github.com/pharmaverse/admiral/blob/562807c34de523a5eb762b6aeb9cdb28da59195b/DESCRIPTION#L69) for admiral. 

NB: you don't need `admiraldev::rdx_roclet` as you don't use the custom roclet.

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+# pharmaverseadam (development version)
+
+<details>
+<summary>Developer Notes</summary>
+
+- Updated the links in the PR template, and updated the `RoxygenNote` field of the DESCRIPTION file. (#160)
```
Comment (manciniedoardo) [inline NEWS.md]:
```suggestion
- Updated the links in the PR template, and upversioned to {roxygen} 8.0.0. (#160)  
```

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Some minor comments

File: DESCRIPTION (hunk 1 above)
Comment (Fanny-Gautier) [reply DESCRIPTION:62]: The `Roxygen check` was failing with the following message:
```
💻 Please rerun the following command on your workstation and push your changes
roxygen2::roxygenize('.')
ℹ roxygen2 version that was used in this workflow: ‘8.0.0’
🙏 Please ensure that the 'RoxygenNote' field in the DESCRIPTION file matches this version
```
`RoxygenNote: 7.3.3` is updated with `Config/roxygen2/version: 8.0.0` while running the command mentioned above.

**Comment by Fanny-Gautier** [comment]
Hi @Lina2689,
Could you please take a look to this PR and provide any comments as necessary?
Thank you.

**Review by Lina2689 — APPROVED** [review]
Looks good!
